packages/cli/src/lib/app-config.ts

```diff
@@ -11,12 +11,18 @@ import { AssertionError, assertObject } from './utils/assert.js';
 import { ResolvedAppPackage } from './app-package.js';
 import deepmerge from 'deepmerge/index.js';
 
+type Endpoints = {
```

Comment:
think this should be `Record<name: string, {url: string, scopes?: string[]}`

it easier to access a record than filtering arrays

Reply:
i totally agree, but thats the format the API requires. Should i ask them to change it?